extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractAgreementPageQuery.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright (c) 2024 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ */
+
+package de.sovity.edc.ext.wrapper.api.ui.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Schema(description = "Filters for querying a Contract Contract Agreement Page")
+public class ContractAgreementPageQuery {
+    @Schema(
+        description = "Optionally filter the resulting contract agreements by their termination status.",
+        requiredMode = Schema.RequiredMode.NOT_REQUIRED)
+    private ContractTerminationStatus statusQuery;
```

Comment:
if we can't agree on the filter vs query thing, could we maybe just call it "terminationStatus" or "status"?

Reply:
yes. Fixed.